meetings/2023-02-22.md

```diff
@@ -0,0 +1,102 @@
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+
+
+### Add TR/2022/notifications-protocol-20221231
+URL: https://github.com/solid/specification/pull/491
+
+* SC: Feedback? Questions?
+
+### Add TR/2022/protocol-20221231
+URL: https://github.com/solid/specification/pull/492
+
+* SC: Feedback? Questions?
+
+
+### Milestone 0.11.0
+URL: https://github.com/solid/specification/milestone/7
+
+* SC: Tentative due date: 2023-04-30.
+
+
+---
+
+* SC: To be picked up next meeting.
+
+### Coordinating further work on authentication and authorization
+* SC: Previous discussion https://github.com/solid/specification/blob/main/meetings/2023-02-15.md#coordinating-further-work-on-authentication-and-authorization
+
+
+
+### Shapes Reuse
+* SC: "Shapes" pertaining to SHACL or ShEX.
+* SC: Should we have a shared space where shapes are developed and reused? Here are some existing work and considerations:
+* SC: Specifications include data modeling, e.g., https://solid.github.io/notifications/protocol#data-model , https://solidproject.org/TR/wac#authorization-rule
+* SC: Specifications describe Constraints and Problem Details, e.g., https://solidproject.org/ED/protocol#constraints-problem-details .
+* SC: Resource servers can process write operations with shape validation.
+* SC: Applications can be informed on shapes and constraints on resources, e.g., https://shapetrees.org/TR/specification/ .
+* SC: Create https://github.com/solid/shapes ? There are services like https://shaperepo.com/ that we can consider to reuse or have something similar that works through Solid org / CG?
+
+
+### Auxiliary Resources with own Access Controls
+URL: https://github.com/solid/specification/issues/501
+
+* SC: About time we have a closer look at this for a future milestone?
+
+
+### Evaluate existing RDF delta/diff formats
+URL: https://github.com/solid/notifications/issues/157
+
+* SC: Let's move this to solid/specification ?
+
+
+
+---
+
+### Topic Proposal
+URL:
+
+* Proposed by [name]
```

Comment:
Lacking quorum, we had a wide ranging conversation that included a number of Solid-related and Solid-tangential topics.  Much food for thought, but not easily captured for minutes.